src/test/kotlin/parts/wisdom/arcraider/ArcTest.kt

```diff
@@ -8,7 +8,8 @@ import java.io.File
 class ArcTest {
     @Test
     fun `load Task from file`() {
-        val file = File(arcData, "training/0a938d79.json")
+        val file = File(arcData, "extend_right.json")
         val task = loadTaskFromFile(file)
+        println(task)
```

Comment:
It's not super useful for a test to print stuff. We should figure out what we expect this to print and do an assertion on it. pro-tip: use the [multiline strings feature](https://realkotlin.com/tutorials/2018-06-26-multiline-string-literals-in-kotlin/) of kotlin

Reply:
For now as long as it parses without crashing I'm happy.